### PR TITLE
fix(state): states are migrated from plannerList to PlannerModule

### PR DESCRIPTION
**Pull request by sanbornsen**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
    * This PR migrate all the states from PlannerListModule to PlannerModule
    * Changes are mostly in `PlannerModule` and `PlannerListModule`
    * All the `listPage` is replaced with `planner` in the state path.

- [ ] What issue/task does this PR references?

- [x] Are the tests Included?
    No. This PR needs no test because it's Module level changes. No feature was added, method was created or modified.
    
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
How come we are approving such a large change without a single test? Especially that the template says it's mandatory.

**Comment by sanbornsen** [comment]
@bartoszmajsak in this PR we changed the path for the test from `listPage` to `planner` thus it looks large. We don't write tests for modules because it only consists of imports and declaration.

**Comment by bartoszmajsak** [comment]
@sanbornsen Then maybe justification on the mandatory step would help to clarify beforehand? Can this be done from now on, for every PR? Otherwise, I would suggest rethinking if we need such a template at all if we don't conform to it.

**Comment by sanbornsen** [comment]
@bartoszmajsak justification on description makes sense. We need this template. A PR like this is very rare.

File: gulpfile.js [hunk 1]
```diff
@@ -89,12 +89,12 @@ mach.transpileLESS = function (src, debug) {
     // paths: [ path.join(__dirname, 'less', 'includes') ]
   }
   return gulp.src(src)
-    .pipe(less({
-      plugins: [autoprefix]
-    }))
     .pipe(lesshint({
       configPath: './.lesshintrc' // Options
     }))
+    .pipe(less({
```
Comment (sahil143) [inline gulpfile.js]: Why the order was changed here for `.pipe(lesshint({` and `.pipe(less({`?

File: src/app/components_ngrx/planner-list/planner-list.module.ts [hunk 2]
```diff
@@ -42,10 +42,6 @@ import { FilterService } from './../../services/filter.service';
 // ngrx stuff
 import { EffectsModule } from '@ngrx/effects';
```
Comment (sahil143) [inline src/app/components_ngrx/planner-list/planner-list.module.ts]: unused imports

File: src/app/components_ngrx/planner-list/planner-list.module.ts [hunk 3]
```diff
@@ -42,10 +42,6 @@ import { FilterService } from './../../services/filter.service';
...
 import { StoreModule } from '@ngrx/store';
```
Comment (sahil143) [inline src/app/components_ngrx/planner-list/planner-list.module.ts]: same as above

**Comment by sanbornsen** [comment]
@bartoszmajsak you can see, even we don't have enough unit tests overall, our functional tests are strong enough to identify problems. Somehow linking work item is broken in this PR and thus the build fails.

**Comment by bartoszmajsak** [comment]
> @bartoszmajsak you can see, even we don't have enough unit tests overall, our functional tests are strong enough to identify problems. Somehow linking work item is broken in this PR and thus the build fails.

@sanbornsen Sure, except that it took ~30minutes to execute :) If you would have faster tests you potentially could spot this problem before opening the PR. Something to consider as an improvement moving forward.

**Comment by sanbornsen** [comment]
@bartoszmajsak sure, couldn't agree more. But situation matters.

**Comment by sudsen** [comment]
[test]

**Comment by sanbornsen** [comment]
[test]

**Pull request merged**